rb/lib/selenium/webdriver/safari/server.rb

```diff
@@ -101,15 +100,15 @@ def wait_for_connection
 
         HEADERS.gsub!("\n", "\r\n")
 
-        HTML = "<!DOCTYPE html><script>#{Safari.resource_path.join('client.js').read}</script>"
```

Comment:
Because Safari.resource_path only exists if we have compiled the safari source. I'd like to be able to download the source and run tests on other browsers without having to do a `./go build` of some kind first.